provision.sh

```diff
@@ -7,5 +7,6 @@ curl -sL https://deb.nodesource.com/setup_0.12 | bash -
 apt-get install -y nodejs
 pip install -r /home/vagrant/ka-lite/requirements_dev.txt
 cd /home/vagrant/ka-lite
+pip install -e .
```

Comment:
We now also install the `kalite` executable globally! So within the VM you can call `kalite` anywhere and it will work.